ci-scripts/install-travis-gcc.sh

```diff
@@ -0,0 +1,21 @@
+
+# Install gcc6 (6.4.0-2 on Mac OS) for Travis builds
+
+if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then
+  sudo apt-get install -qq g++-6 && sudo update-alternatives --install /usr/bin/g++ g++ /usr/bin/g++-6 90;
+fi
+
+if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then
```

Comment:
@gz-c this issue can be solved like this , or instead Travis -specific checks can be returned back to `.travis.yml` and only the lines that follow would remain in this script .